core/src/main/java/org/hibernate/ogm/type/impl/TypeTranslatorImpl.java

```diff
@@ -121,4 +135,40 @@ else if ( type instanceof org.hibernate.type.CollectionType ) {
 		}
 		throw log.unableToFindGridType( type.getClass().getName() );
 	}
+
+	/*
+	   We directly pass the converter "database" type as value to the Tuple
```

Comment:
Actually, is this really the behavior we want ?
Don't we rather want the user to have final say on how the type is transformed?

I guess it is always fine to pass the guessed Type into the `TypeTranslatorImpl.getType()` again. @gunnarmorling can you see downsides where it would be inappropriate?